AmazonChimeSDK/AmazonChimeSDK/audiovideo/audio/activespeakerdetector/DefaultActiveSpeakerDetector.swift

```diff
@@ -16,12 +16,14 @@ typealias DetectorCallback = (_ attendeeIds: [AttendeeInfo]) -> Void
 @objcMembers public class DefaultActiveSpeakerDetector: ActiveSpeakerDetectorFacade, RealtimeObserver {
     private static let activityWaitIntervalMs = 1000
     private static let activityUpdateIntervalMs = 200
```

Comment:
does this not need to update?

Reply:
Our `IntervalScheduler` takes interval in milliseconds, so changing that would be a breaking change.
So to reduce impact, those are not changed, and only the internal calculation inside `DefaultActiveSpeakerDetector ` is changed.